src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-form/pool-form.component.scss

```diff
@@ -0,0 +1,8 @@
+@use './src/styles/vendor/variables' as vv;
+
+.warning-icon {
+  align-items: center;
+  color: vv.$warning;
+  margin-bottom: 5px;
+  margin-left: 3px;
+}
```

Comment:
Optional comment: If we find this is something 're-usable' from other parts of the code (and it looks so given the password expiration PR proposed kinda such a thing, right?), why not moving this to a top-level SCSS, instead of per-component one?

Reply:
Done